uhabits-android/src/main/java/org/isoron/uhabits/utils/ViewExtensions.kt

```diff
@@ -101,6 +112,32 @@ fun ViewGroup.buildToolbar(): Toolbar {
     return inflater.inflate(R.layout.toolbar, null) as Toolbar
 }
 
+fun ViewGroup.buildKonfettiView(): View {
+    val inflater = LayoutInflater.from(context)
+    return inflater.inflate(R.layout.konfetti, null) as View
+}
+
+fun showConfetti(view: View) {
+    val viewId = R.id.konfetttiView
```

Comment:
```suggestion
    val viewId = R.id.konfettiView
```
Is this a typo or is there a reason to have 3 `t`?